### ISSUE TITLE
[Accessibility][Object Information - Property Pane_Connect to QnA Knowledgebase]: No info is provided for the 'Enter' button to perform action.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2130]
**User Experience** 
The screen reader user will get confused if the proper information is not announced by the narrator.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and turn on narrator.
2. Navigate to already created dialog tree item and activate it. Such as- Greetings, Actions, EndDialog etc.
3. Click on "+" and select 'Access External resources'. Then select 'Connect to QnA Knowledgebase' from the available options.
4. Navigate to edit fields under "Strict Filters' having functionality to enter.
5. Observer the issue in Property Pane.

**Actual:** 
The screen reader didn't announce any information about the functionality of 'Enter' to insert the data. Same issue found for the 'Value and Name' edit field.
Narrator only narrates the placeholder whereas the edit fields have functionality that if user press 'Enter' the data  gets inserted. 

**Expected:**
The narrator should announce the functionality of 'Enter'. It should announce as 'Press Enter to insert name or press tab to move the cursor on the next interactive element'. 
Similarly for value edit field 'Press Enter to insert value or press tab to move the cursor on the next interactive element'. 

**Note:**
Issue is observed for all the Property Pane which have similar control such as 'Property Pane_Ask a question (Other)'.

**MAS Impacted:** [MAS4.2.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858})

**Attachment:**
[No info is provided for the 'Enter' button to perform action.zip](https://github.com/microsoft/BotFramework-Composer/files/4273495/No.info.is.provided.for.the.Enter.button.to.perform.action.zip)
![Narrator should announce the functionality of Enter Key](https://user-images.githubusercontent.com/60913131/75646577-1e242800-5c70-11ea-8b21-52ab0e0c24d4.png)

**Comment by corinagum** [comment]
@ashish315 just FYI, looks like the attachment didn't make it onto this issue. :)

**Comment by ashish315** [comment]
@corinagum The attachments are updated now.

**Comment by ashish315** [comment]
@beyackle As checked on below environment the issue is still repro.

Issue repro at-
1. "Validation rules" edit field at 'Property Pane_Ask a question (Other tab)'.
2. "Name" edit field at 'Property Pane_Connect to QnA Knowledgebase'.

**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
Browser: Microsoft Edge Canary {Version 83.0.474.0 (Official build) canary (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Comment by corinagum** [comment]
As I understand the filed issue, I am seeing this as fixed. (I have a small nit, detailed below)

@ashish315 In reference to your repro steps above ("Validation rules..."), could you provide a video of this behavior?

![image](https://user-images.githubusercontent.com/14900841/78836359-c67f8780-79a6-11ea-9917-eb5c51e0429c.png)
Instead of field, should this say something like 'to advance to the value input'?

**Comment by ashish315** [comment]
The narrator should announce 'Press Enter to insert name' while the focus is at name edit field.
Please look into the attached video: 
[Regress 2130.zip](https://github.com/microsoft/BotFramework-Composer/files/4456363/Regress.2130.zip)

**Comment by beyackle** [comment]
I'll handle this. It's just a message change.

**Comment by ashish315** [comment]
@beyackle  As checked and verified on below environment, the issue is not fixed. I downloaded below Master build file last updated on 18th April.

**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
![image](https://user-images.githubusercontent.com/60913131/79755879-6c33de80-8337-11ea-90cd-8dd51ac56600.png)

**Comment by beyackle** [comment]
Okay, I've gone back and double-checked this, and while it does correctly read the caption when the field is entered, there's still a minor bug where entering a name into the first part of an object field and then hitting Enter without entering a value will advance to a new row (as intended), but then the SR reads nothing out for the new row. So while this issue is mostly fixed, that one bug still remains.

**Comment by ashish315** [comment]
As checked the issue is fixed now, so HCL is closing the issue.
